src/pretix/control/views/event.py

```diff
@@ -1040,6 +1040,8 @@ class EventLog(EventPermissionRequiredMixin, PaginationMixin, ListView):
     context_object_name = 'logs'
 
     def get_queryset(self):
+        # technically, we'd also need to sort by pk since this is a paginated list, but in this case we just can't
+        # bear the performance cost
```

Comment:
See comment below regarding replacing the datetime-index with a (datetime, id)-index.